ui/page/start_page.go

```diff
@@ -121,6 +122,7 @@ func (sp *startPage) HandleUserInteractions() {
 					m.Dismiss()
 
 					sp.ChangeWindowPage(NewMainPage(sp.Load), false)
+					sp.ChangeFragment(privacy.NewSetupPrivacyPage(sp.Load, wal))
```

Comment:
Having this here does not make sense to me.

this should be on the main_page.go file as the window switches to the mainpage after successful setup.
even if this may be working, it is bad logic/flow.